graphql/tests/query/adminSettings/data.json

```diff
@@ -42,8 +42,8 @@
         "keyset_url": "https://kanopy.com/keyset"
       },
       "multi": {
-        "read_write": ["admin", "privileged"],
-        "read_only": ["admin", "privileged", "regular"]
+        "read_write": ["okta"],
+        "read_only": ["github"]
```

Comment:
What is this change? 👀

Reply:
the previous values were not "valid" so i just updated to a real one haha